src/test/java/com/thealgorithms/sorts/WaveSortTest.java

```diff
@@ -0,0 +1,34 @@
+package com.thealgorithms.sorts;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import java.util.stream.Stream;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.MethodSource;
+
+public class WaveSortTest {
+    @ParameterizedTest
+    @MethodSource("arraysToWaveSort")
+    public void waveSortTest(Integer[] array) {
+        WaveSort waveSort = new WaveSort();
+        assertTrue(waveSort.isWaveSorted(waveSort.sort(array)));
+    }
```

Comment:
Imagine the situation when `sort` method is changed to one returning an empty array. Since empty array is _wave sorted_ this test will pass. Could you please rewrite this test in such a way, that such situation would be excluded?

For example, one could:
- compute the _histogram_ of the input array (i.e. a hash map with number of occurrences of each element of the input array),
- compute the _histogram_ of the sorted one,
- check if they are equal.

Reply:
Fixed